doc/source/raysgd/raysgd_ft.rst

```diff
@@ -0,0 +1,31 @@
+RaySGD Fault Tolerance
+======================
+
+.. note:: Fault tolerance is currently only enabled for the PyTorchTrainer.
+
+For distributed deep learning, jobs are often run on infrastructure where nodes can be pre-empted frequently (i.e., spot instances in the cloud). To overcome this, RaySGD provides **fault handling** features that enable training to continue regardless of node failures.
+
+.. code-block:: bash
+
+    trainer.train(max_retries=N)
+
+
+How does it work?
```

Comment:
After reading this I'm left wondering where the trainer runs and what happens if it fails. Maybe you can add a few sentences addressing that?

Reply:
done!